Formula/l/llvm.rb

```diff
@@ -454,11 +435,17 @@ def caveats
   end
 
   test do
-    llvm_version = Version.new(Utils.safe_popen_read(bin/"llvm-config", "--version").strip)
-    soversion = llvm_version.major.to_s
-    soversion << "git" if head?
+    llvm_version = Utils.safe_popen_read(bin/"llvm-config", "--version").strip
+    soversion = Version.new(llvm_version).major.to_s
+
+    if llvm_version.end_with?("git")
+      soversion << "git"
+    elsif llvm_version.end_with?("rc")
+      assert_equal version.major_minor_patch, llvm_version.major_minor_patch
```

Comment:
Now RC versions are suffixed `rc` without the RC number. See llvm/llvm-project#64225.